README.adoc

```diff
@@ -178,34 +206,61 @@ configuration to log in to view the data.
 ----
 
 InventoryManager.java
-[source, java, linenums, role='code_column']
+[source, java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[tags=InventoryManager]
+include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[]
 ----
 
-Apply the [hotspot=23-26]`@Timed` annotation to the [hotspot=27-29]`get()` method to track how frequently the method is
-invoked and also how long it takes for the invocation to complete. This annotation has these
-metadata fields:
+Apply the [hotspot=timedForGet]`@Timed` annotation to the [hotspot=get]`get()` method,
+and apply the [hotspot=timedForList]`@Timed` annotation to the [hotspot=list]`list()` method.
+
+This annotation has these metadata fields:
 
 |===
-|[hotspot=23]`name` | Optional. Use this field to name of the metric.
-|[hotspot=24]`absolute` | Optional. Use this field to determine whether the metric name is the exact name that is specified in the [hotspot=23]`name` field or that is specified with the package prefix.
-|[hotspot=25-26]`description` | Optional. Use this field to describe the purpose of the metric.
+|[hotspot=nameForGet hotspot=nameForList]`name` | Optional. Use this field to name the metric.
+|[hotspot=tagForGet hotspot=tagForList]`tags` | Optional. Use this field to add tags to the metric with the same [hotspot=nameForGet hotspot=nameForList]`name`.
+|[hotspot=absoluteForGet hotspot=absoluteForList]`absolute` | Optional. Use this field to determine whether the metric name is the exact name that is specified in the [hotspot=nameForGet hotspot=nameForList]`name` field or that is specified with the package prefix.
+|[hotspot=desForGet hotspot=desForList]`description` | Optional. Use this field to describe the purpose of the metric.
 |===
 
-Apply the [hotspot=42-44]`@Counted` annotation to the [hotspot=45-47]`list()` method to count how many times the
-http://localhost:9080/inventory/systems[http://localhost:9080/inventory/systems^] URL is accessed. 
+Both the [hotspot=get]`get()` and [hotspot=list]`list()` methods are annotated with the `@Timed` metric and have the same [hotspot=nameForGet hotspot=nameForList]`inventoryProcessingTime` name. The `@Timed` annotation tracks how frequently and how long it takes for the method invocation to complete. The [hotspot=tagForGet]`method=get` and [hotspot=tagForList]`method=list` tags add a dimension to the collected metric data for the inventory processing time in getting the system properties.
+
+- The `method=get` processing time tag indicates the time to get the system properties from calling the `system` service.
+- The `method=list` processing time tag indicates the time for the `inventory` service to list all of the system properties in the inventory.
```

Comment:
Sounds a little bit weird to me, sounds like we're saying the tag itself is indicating time?

Reply:
Fixed